clang/unittests/Format/ConfigParseTest.cpp

```diff
@@ -1294,11 +1294,14 @@ TEST(ConfigParseTest, HandleNonCppDotHFile) {
 
   Style = {};
   Style.Language = FormatStyle::LK_Cpp;
-  EXPECT_EQ(parseConfiguration("Language: ObjC", &Style,
```

Comment:
Why remove this?

Reply:
I've added the explanation in the commit message. Please see https://github.com/llvm/llvm-project/pull/159908#issue-3436689745.